packages/react-server/src/ReactFizzServer.js

```diff
@@ -1911,11 +1911,6 @@ function flushCompletedQueues(
     }
     completedBoundaries.splice(0, i);
 
-    // Allow anything written so far to flush to the underlying sink before
-    // we continue with lower priorities.
-    completeWriting(destination);
```

Comment:
This would prevent Node.js from flushing the already completed parts. I was going to split this so that this gets put into a timer based throttle anyway since flushing the partial work early isn't always better.

Reply:
Ah, should I split implementations so we get buffer based flushing in Browser only?

I'm not sure what you had in mind for timer based throttling but I can leave the node version functionally identical

Reply:
Lmk how you feel about the requestFlush approach for restoring the Node behavior